src/plots/Volume/avtVisItVTKRenderer.C

```diff
@@ -640,12 +644,69 @@ avtVisItVTKRenderer::UpdateRenderingState(vtkDataSet * in_ds,
         }
         else
 #endif
+    }
```

Comment:
Looks like the `else` on line 645 should be removed.

Reply:
Good catch. `else` removed